distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso

```diff
@@ -1088,6 +1089,19 @@ type Column
             _           -> False
         apply_unary_operation self IsInfiniteOperation.INSTANCE new_name if_unsupported=(apply_unary_map self new_name fallback Value_Type.Boolean)
 
+    ## GROUP Standard.Base.Math
+       ICON math
+       Returns a column of booleans, with `True` items at the positions where
+       this column contains a non-infinite, non-NaN floating point value. This
+       is only applicable to double columns.
+    is_finite : Column
+    is_finite self = Value_Type.expect_numeric self <|
+        new_name = naming_helper.function_name "is_finite" [self]
+        fallback x = case x of
+            _ : Float   -> x.is_finite
+            _           -> False
```

Comment:
```suggestion
            _ : Float   -> x.is_finite
            _ : Number -> True
            _ : Decimal -> True
            _           -> False
```

Reply:
Actually changing to:
```
            _ : Float   -> x.is_finite
            _           -> True
```
Because we're already using `expect_numeric`, and only Float can be infinite.

Reply:
But this method is `is_finite` and so shouldn't default be true?